client/app/account/user/security/users/users.html

```diff
@@ -30,38 +36,35 @@ <h1 data-translate="user_users_title"></h1>
                 <thead>
                     <tr>
                         <th data-translate="user_users_login" scope="col"></th>
-                        <th data-translate="user_users_full_login" scope="col"></th>
-                        <th data-translate="user_users_email" scope="col"></th>
+                        <th data-translate="user_users_role" scope="col"></th>
                         <th data-translate="user_users_creation_date" scope="col"></th>
-                        <th data-translate="user_users_last_update_date" scope="col"></th>
+                        <th data-translate="user_users_last_password_update_date" scope="col"></th>
                         <th data-translate="user_users_status" scope="col"></th>
```

Comment:
+`colspan="2"` and then delete the empty `<th>` right after.

Reply:
Done